scss/atoms/typography/_typography.scss

```diff
@@ -44,6 +44,24 @@ h6 {
 	@include heading($h6);
 }
 
+p {
+	text-align: map-get($p,text-alignment);
+	margin: em(map-get($p,margin-top)) em(map-get($p,margin-right)) em(map-get($p,margin-bottom)) em(map-get($p,margin-left));
+	line-height: 1.5;
+	font-size: map-get($font-default, font-size);
```

Comment:
You are assigning the default font size to p directly. That will happen anyways as p will inherit the style.
Instead you should have a font-size variable in $p map which is set to the default font size be default, and user can overwrite if he wants